__tests__/server.test.js

```diff
@@ -40,3 +40,37 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET:200 responds with article by its id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(typeof article.author).toBe("string");
+        expect(typeof article.title).toBe("string");
+        expect(typeof article.article_id).toBe("number");
```

Comment:
This needs to be the number you have requested, as at the moment, this could match any and you wouldnt know you've got the correct one back